pyproject.toml

```diff
@@ -106,5 +106,9 @@ skan = [
 [tool.setuptools_scm]
 write_to = 'src/skan/_version.py'
 
+[tool.pytest.ini_options]
+minversion = "7.4.2"
+addopts = "-W ignore"
```

Comment:
Why did you add this? (ie what warnings are we ignoring here and is that wise 😅)

Reply:
Currently its our own deprecation warnings that were recently added for the removal of `_`.

```bash
src/skan/test/test_csr.py:407
  /home/neil/work/git/hub/ns-rse/skan/src/skan/test/test_csr.py:407: VisibleDeprecationWarning: separator in column name will change to _ in version 0.13; to silence this warning, use `separator='-'` to maintain current behavior and use `separator='_'` to switch to the new default behavior.
    csr.summarize(csr.Skeleton(skeleton1)),
```

Happy to remove this though I only added it so it was easier to read test output whilst writing them.

Reply:
I'll leave it for now and make an issue to either wrap those in pytest.warns or pass the separator arg.